modules/engage-ui/src/main/java/org/opencastproject/engage/ui/PlayerRedirect.java

```diff
@@ -51,6 +53,15 @@
   title = "Configurable Player Endpoint",
   abstractText = "This service redirects to configured players.",
   notes = {})
+@Component(
+  immediate = true,
+  service = PlayerRedirect.class,
+  property = {
+    "service.description=Configurable Player Endpoint",
+    "opencast.service.type=org.opencastproject.engage.ui.player.redirect",
+    "opencast.service.path=/play"
+  }
+)
```

Comment:
Looking at the surrounding code, this should be

```suggestion
  })
```

Reply:
If you want it, you get it. Fixed.